packages/select/src/components/select/suggest.tsx

```diff
@@ -284,6 +285,23 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
         Utils.safeInvokeMember(this.props.popoverProps, "onOpened", node);
     };
 
+    private handlePopoverClosed = (node: HTMLElement) => {
```

Comment:
nit: we try to avoid the term "node" except when talking about Tree nodes... so please rename this to `element`